tests/test_transformseq.py

```diff
@@ -400,47 +400,48 @@ def test_invalid_dimensions(self):
 
     def test_PlainTransforms_invalid_todims(self):
         with self.assertRaisesRegex(ValueError, 'expected transforms with todims=2, but got .*'):
-            nutils.transformseq.PlainTransforms([(x1, c0), (x1, c1)], 2, 1)
+            nutils.transformseq.PlainTransforms(((x1, c0), (x1, c1)), 2, 1)
 
     def test_PlainTransforms_invalid_fromdims(self):
         with self.assertRaisesRegex(ValueError, 'expected transforms with fromdims=2, but got .*'):
-            nutils.transformseq.PlainTransforms([(x2, e00), (x2, e11)], 2, 2)
+            nutils.transformseq.PlainTransforms(((x2, e00), (x2, e11)), 2, 2)
 
     def test_PlainTransforms_multiple_fromdims(self):
         with self.assertRaisesRegex(ValueError, 'expected transforms with fromdims=2, but got .*'):
-            nutils.transformseq.PlainTransforms([(x2, c00, e00), (x2, c01, i20, i20)], 2, 2)
+            nutils.transformseq.PlainTransforms(tuple(transform.canonical((x2, *tail)) for tail in ((c00, e00), (c01, i20, i20))), 2, 2)
+            nutils.transformseq.PlainTransforms(tuple(transform.canonical((x2, *tail)) for tail in ((c00, e00), (c01, i20, i20))), 2, 2)
```

Comment:
I don't think we every reach the second line.

Reply:
Reverted all changes to transformsseq in light of the plan to make sequences non-singleton.